This patch is one of the `ndk-sysroot` from termux main repo.
--- a/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/c++/v1/cmath
+++ b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/c++/v1/cmath
@@ -301,7 +301,7 @@
 #include <__type_traits/remove_cv.h>
 #include "version"
 
-#include <math.h>
+#include <c++/v1/math.h>
 
 #ifndef _LIBCPP_MATH_H
 #   error <cmath> tried including <math.h> but didn't find libc++'s <math.h> header. \
--- a/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/c++/v1/math.h
+++ b/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/c++/v1/math.h
@@ -297,9 +297,7 @@
 #  pragma GCC system_header
 #endif
 
-#  if __has_include_next(<math.h>)
-#    include_next <math.h>
-#  endif
+#include "../../math.h"
 
 #ifdef __cplusplus
 
